README.md

```diff
@@ -9,30 +9,31 @@ The remainder of the charts in the response can be produced from code in the rep
 - Meet the data science cookiecutter [requirements](http://nestauk.github.io/ds-cookiecutter/quickstart), in brief:
   - Install: `direnv` and `conda`
 - Clone the repo: `git clone git@github.com:nestauk/asf_welsh_energy_consultation.git`
-- Navigate to the repo folder
+- Navigate to your local repo folder
 - Checkout the correct branch if not working on dev
-- Run `make install` to configure the development environment:
+- Run `direnv allow`
+- Run `make install` to configure the development environment. This will:
   - Setup the conda environment
   - Configure `pre-commit`
-- Run `direnv allow`
+  - Install packages listed in `requirements.txt`
 - Activate conda environment: `conda activate asf_welsh_energy_consultation`
-- Install requirements: `pip install -r requirements.txt`
-- Download the data:
+- Run `make inputs-pull` to pull the zipped static data from S3 and put it in `/inputs`
+- Follow the instructions (here)[https://github.com/altair-viz/altair_saver/#additional-requirements] to install
+  ChromeDriver, node, and the required packages into your conda environment.
 
-  - `make inputs-pull` will pull the zipped data from S3 and put it in `/inputs`
-  - Download `mcs_installations_230315.csv` and `mcs_installations_epc_full_230315.csv` from `asf-core-data` S3 bucket (they can be found under `outputs/MCS/`) and add them to `inputs/data/`- Perform additional setup in order to save plots:
+- Run `python asf_welsh_energy_consultation/analysis/produce_plots.py --local_data_dir <YOUR_LOCAL_DIR>`. You need to specify the path to the local
+  directory where your local copy of the EPC data is/will be saved by replacing `<YOUR_LOCAL_DIR>` with the path. You can specify which
```

Comment:
Wondering if you should say something more about the local directory something like (or similar):

_If you haven't created a local directory for ASF core data, you can create a folder in your Documents called "ASF data"_

Reply:
good idea